recipes/plotly-resampler/meta.yaml

```diff
@@ -1,53 +0,0 @@
-{% set name = "plotly-resampler" %}
```

Comment:
This file shouldn't be modified by this PR.

Reply:
very strange...

Reply:
@dopplershift I think I fixed it?